ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx

```diff
@@ -24,6 +24,8 @@ export interface PodLogsProps {
     page: {number: number; untilTimes: string[]};
     timestamp?: string;
     setPage: (pageData: {number: number; untilTimes: string[]}) => void;
+    containerGroups?: any[];
```

Comment:
We need to load container groups in fullscreen logs viewer: https://github.com/argoproj/argo-cd/blob/master/ui/src/app/applications/components/application-fullscreen-logs/application-fullscreen-logs.tsx 


Otherwise fullscreen view has containers menu but menu is emplty

Reply:
Fullscreen view did not have an option to switch between containers, so we don't have to replicate it. Instead of populating list of container groups we can just hide the container menu in fullscreen.

Reply:
Thanks @alexmt for the catch! I didn't know about this fullscreen logs viewer. Issue fixed.